cnxarchive/tests/test_views.py

```diff
@@ -989,6 +994,7 @@ def test_exports_without_version(self):
         self.assertEqual(cm.exception.headers,
                          [('Location', '/exports/{}@5.pdf'.format(id))])
 
+    @mock.patch('cnxarchive.utils.fromtimestamp', mock.Mock(side_effect=testing.mocked_fromtimestamp))
```

Comment:
This works when I try to run the tests individually.  It fails when I'm running all the tests in ViewsTestCase.

Doing this instead works locally for me: `@mock.patch('cnxarchive.views.fromtimestamp', mock.Mock(side_effect=testing.mocked_fromtimestamp))` cnxarchive.**views**.fromtimestamp instead of cnxarchive.utils.fromtimestamp.

May be it's because we import just the function `fromtimestamp` into `cnxarchive.views` from `cnxarchive.utils` instead of importing `cnxarchive.utils` and doing `cnxarchive.utils.fromtimestamp`.